include/net/lwm2m_client_utils.h

```diff
@@ -68,6 +68,42 @@ struct modem_mode_change {
  */
 int lwm2m_init_security(struct lwm2m_ctx *ctx, char *endpoint, struct modem_mode_change *mmode);
 
+/**
+ * @brief Set security object to PSK mode.
+ *
+ * Any pointer can be given as a NULL, which means that data related to this field is set to
+ * zero legth in the engine. Effectively, it causes that relative data is not written into a modem.
+ * This can be used if the given data is already provisioned to modem.
```

Comment:
```suggestion
 * zero legth in the engine. Effectively, it causes that relative data is not written into the modem.
 * This can be used if the given data is already provisioned to the modem.
```